axelrod/tests/strategies/test_selfsteem.py

```diff
@@ -20,19 +20,38 @@ class TestSelfSteem(TestPlayer):
     }
 
     def test_strategy(self):
-        # Check for f > 0.95
-        self.responses_test([D], [C] * 2 , [C] * 2)
-        self.responses_test([D], [C] * 13, [C] * 13)
-
-        # Check for f < -0.95
-        self.responses_test([C], [C] * 7, [C] * 7)
-        self.responses_test([C], [C] * 18, [D] * 18)
-
-        # Check for -0.3 < f < 0.3
-        self.responses_test([C], [C] * 20, [C] * 20, seed=6)
-        self.responses_test([D], [C] * 20, [D] * 20, seed=5)
-
-        # Check for 0.95 > abs(f) > 0.3
-        self.responses_test([C], [C], [C])
-        self.responses_test([D], [C] * 16, [D] * 16)
-        self.responses_test([C], [D] * 9, [C] * 9)
+         
+            # Check for f > 0.95, defect
+            actions = [(C, C), (C, C), (D, C), (D, C)]
+            self.versus_test(axelrod.Cooperator(), expected_actions = actions)
```

Comment:
Could you change `expected_actions = actions` to `expected_actions=actions` (no space either side of `=`).

This is a PEP8 thing.